packages/features/ee/organizations/pages/settings/attributes/attributes-list-view.tsx

```diff
@@ -0,0 +1,182 @@
+  MULTI_SELECT: "Multi-select",
+};
+
+function AttributeItem({ attribute }: { attribute: AttributeItemProps }) {
+  const utils = trpc.useUtils();
+  const [isEnabled, setIsEnabled] = useState(attribute.enabled);
+  const mutation = trpc.viewer.attributes.toggleActive.useMutation({
+    onSuccess: () => {
+      showToast(t("attribute_updated_successfully"), "success");
+    },
+    onError: (err) => {
+      showToast(err.message, "error");
+    },
+  });
+
+  const deleteMutation = trpc.viewer.attributes.delete.useMutation({
+    onSuccess: () => {
+      showToast(t("attribute_deleted_successfully"), "success");
+      utils.viewer.attributes.list.invalidate();
+    },
+    onError: (err) => {
+      showToast(err.message, "error");
+    },
+  });
+
+  const handleToggle = (checked: boolean) => {
+    // Optimistically update the local state
+    setIsEnabled(checked);
+    mutation.mutate(
+      { attributeId: attribute.id },
+      {
+        onError: (err) => {
+          setIsEnabled(!checked);
+          showToast(err.message, "error");
+        },
+      }
+    );
+  };
+
+  const handleDelete = () => {
+    deleteMutation.mutate({ id: attribute.id });
+  };
+
+  const { t } = useLocale();
+  return (
+    <ul className="focus-within:border-emphasis flex justify-between p-4" key={attribute.id}>
+      <div>
+        <h3 className="leadning-none text-sm font-semibold">{attribute.name}</h3>
+        <p className="text-default leadning-none inline-flex items-center gap-1 text-sm font-normal">
+          {TypeToLabelMap[attribute.type]}
+          {attribute.options?.length > 0 && (
+            <>
+              <span className="text-muted">•</span>
+              <span>{attribute.options.length} options</span>
+            </>
+          )}
+        </p>
+      </div>
+      <div className="flex gap-4">
+        <Switch checked={isEnabled} onCheckedChange={handleToggle} />
+        <Dropdown modal={false}>
+          <DropdownMenuTrigger asChild>
+            <Button
+              type="button"
+              variant="icon"
+              color="secondary"
+              StartIcon="ellipsis"
+              className="ltr:radix-state-open:rounded-r-md rtl:radix-state-open:rounded-l-md"
+            />
+          </DropdownMenuTrigger>
+          <DropdownMenuContent className="min-w-[200px]">
+            <DropdownMenuItem>
+              <DropdownItem
+                type="button"
+                StartIcon="pencil"
+                href={`/settings/organizations/attributes/${attribute.id}/edit`}>
+                {t("edit")}
+              </DropdownItem>
+            </DropdownMenuItem>
+            <DropdownMenuItem>
+              <DropdownItem
+                type="button"
+                StartIcon="trash-2"
+                color="destructive"
+                disabled={deleteMutation.isPending}
+                onClick={handleDelete}>
+                {t("delete")}
+              </DropdownItem>
```

Comment:
should we have a modal to confirm deletion?

Reply:
@ciaranha also here

Reply:
Does that relate to this?

![CleanShot 2024-08-09 at 09 15 05@2x](https://github.com/user-attachments/assets/307a9d5e-decb-475e-a4c1-ed2b8043470c)

Reply:
Oh i missed this in designs lol - i can do this

Reply:
Pushed: 
![CleanShot 2024-08-09 at 10 20 46](https://github.com/user-attachments/assets/c4612fd0-48fe-4b74-bcd0-e920dfd36574)